src/ess/sans/reduction.py

```diff
@@ -17,19 +22,22 @@ def reduce_to_q(data, *, q_bins, reducer, wavelength_bands=None):
     Example:
     >>> reduced = reduce_to_q(data, q_bins=q_bins, reducer=simple_reducer('spectrum'))  # noqa: E501
     """
-    # TODO Backup of the coord is necessary until `convert` can keep original
     wavelength = data.coords['wavelength']
     data = scn.convert(data, 'wavelength', 'Q', scatter=True)
     if wavelength_bands is None:
         data = sc.histogram(data, q_bins)
         return reducer(data)
+
+    wavelength.rename_dims({'wavelength': 'Q'})
     data.coords['wavelength'] = wavelength
     bands = None
+
     for i in range(wavelength_bands.sizes['wavelength'] - 1):
         low = wavelength_bands['wavelength', i]
         high = wavelength_bands['wavelength', i + 1]
-        band = sc.histogram(data['wavelength', low:high], q_bins)
+        band = sc.histogram(data['wavelength', low:high].copy(), q_bins)
```

Comment:
```suggestion
        band = sc.histogram(data['wavelength', low:high], q_bins)
```
`sc.histogram` does not modify the input, no need to copy.